yelp_rca.ipynb

```diff
@@ -0,0 +1,255 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "text_train_file = \"/share/shubh/data/yelp_polarity/train_yelp.csv\"\n",
+    "text_test_file = \"/share/shubh/data/yelp_polarity/test_yelp.csv\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Training a text classifier using Universal Deep Transformers is just a few lines of code.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "======================= Bolt Model =======================\n",
```

Comment:
run in release mode, we dont want output to show model details
for now remove cell output